torch/_dynamo/symbolic_convert.py

```diff
@@ -2261,19 +2261,13 @@ def inline_call(cls, parent, func, args, kwargs):
             return cls.inline_call_(parent, func, args, kwargs)
 
     @staticmethod
-    def check_inlineable(func):
+    def check_inlineable(func) -> skipfiles.SkipResult:
```

Comment:
This approach looks good to me, but I have a pending work to merge allow_in_graph/inline/skip check into the unified ```trace_rules.lookup```. I'll move all of the ```check_inlineable``` logics into that function. Then we'll wrap a python function as ```UserFunctionVariable``` only if ```trace_rules.lookup``` tells us to inline it, and wrap as ```SkipFilesVariable``` if ```trace_rules.lookup``` tells us to skip it. I think it would help us make code more concise. I had the draft PR locally, but there are several trivial issues need to fix. Feel free to merge this PR if it blocks your work, I can rebase when I submit my PR.

Reply:
How long do you think until your PR will be ready?

Reply:
Next week

Reply:
I can probably wait then